src/Model/Model_PDF.php

```diff
@@ -231,17 +240,98 @@ public function process_pdf( $pid, $lid, $action = 'view' ) {
 			return $middleware;
 		}
 
-		/* Add backwards compatibility support for certain settings */
-		$settings = $this->apply_backwards_compatibility_filters( $settings, $entry );
+		/*
+		 * Normalize the PDF action
+		 * The PDF cache introduced in 6.12 relies on a hash generated from the form, entry, and pdf settings
```

Comment:
@jakejackson1  Add more info about editing the form, entry or PDF settings auto-expires a PDF so the document is always fresh.